expo/expo
# [docs] Remove custom server and add CORS headers to Next

# Why

This removes the custom `server.js` we had as workaround for both [CORS headers](https://nextjs.org/docs/api-reference/next.config.js/headers) and [trailing slashes](https://nextjs.org/docs/api-reference/next.config.js/trailing-slash), both replaced by built-in support.

# How

- Looked at the `server.js` and replaced its functionality with built-in support.

# Test Plan

- Run `$ yarn dev`
- Check if `404` pages are rendered properly in dev mode
- Check if both with and without trailing slashes are handled correctly
- Check if static files, like images or Expo logo, are handled property
